### PR TITLE
Psg 4901 use repo templates

### PR DESCRIPTION
**Pull request by jenn-macfarlane**
## What's New?

Uses repo templates to format README, LICENSE, and remove community files in favour of using organization community files.

> [!NOTE]  
> This repo contains documentation we do not yet have in our docs platform - I've integrated it as best I could to keep it somewhat in line with the template and preserving details.

## Screenshots (if appropriate):

CleanShot 2024-10-22 at 15 42 00@2x

CleanShot 2024-10-22 at 15 42 09@2x

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Documentation update

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have manually tested my code thoroughly
- [ ] I have added/updated inline documentation for public facing interfaces if relevant
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing integration and unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

## Additional context

Add any other context or screenshots about the pull request here.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,81 +1,101 @@
...
 
-```
-npm i --save @passageidentity/passage-flex-react-native
-```
+<p>
+ You'll need a free Passage account and a Passkey Flex app set up in <a href="https://console.passage.id/">Passage Console</a> to get started. <br />
+ <sub><a href="https://docs.passage.id/home#passage-console">Learn more about Passage Console →</a></sub>
+</p>
 
+Add an Android and/or iOS app in the Native Apps section.
```
Comment (jenn-macfarlane) [inline README.md:39]: I think it makes sense to pull up these more detailed prereq steps into this section. We can delete them/edit what we have in the actual html template when we remove after we have this enshrined in docs.

File: README.md [hunk 2]
```diff
@@ -1,81 +1,101 @@
...
+```
+npx expo prebuild
+```
 
-### Bare React Native
+#### Bare React Native
 
 See our [Passkey Complete documentation](https://docs.passage.id/complete/cross-platform-passkey-configuration/cross-platform-ios-configuration) for setting up a React Native app for passkeys and Passage.
 
+#### Hosted associated domains files
```
Comment (jenn-macfarlane) [inline README.md:74]: This felt right to put in this section as well

File: README.md [hunk 3]
```diff
@@ -1,81 +1,101 @@
...
-
+### Initialize
 ```tsx
-const passageFlex = new PassageFlex('YOUR_APP_ID');
+const passageFlex = new PassageFlex('YOUR_PASSAGE_APP_ID');
 ```
 
+### Go Passwordless
+
+Find all more details about Passkey Flex on our [Passkey Flex Documentation](https://docs.passage.id/flex) page.
```
Comment (jenn-macfarlane) [inline README.md:95]: Because we don't have docs, figured I'd link to the Flex docs homepage.

File: README.md [hunk 4]
```diff
@@ -1,81 +1,101 @@
...
+
+# API Reference
```
Comment (jenn-macfarlane) [inline README.md:97]: Follows what we do in passage-react in the web monorepo

**Comment by agebhardt** [comment]
I really like the approach here! I also added a suggestion via commit, @jenn-macfarlane! Open to your thoughts, but my goal was to simplify the initial view of the README a bit more 😄

**Comment by jenn-macfarlane** [comment]
@agebhardt looks good to me!

**Review by agebhardt — APPROVED** [review]

**Pull request merged**